cpp/ql/lib/semmle/code/cpp/security/InvalidPointerDereference/AllocationToInvalidPointer.qll

```diff
@@ -176,29 +172,29 @@ private module Config implements ProductFlow::StateConfigSig {
   class FlowState2 = int;
 
   predicate isSourcePair(
-    DataFlow::Node source1, FlowState1 state1, DataFlow::Node source2, FlowState2 state2
+    DataFlow::Node allocSource, FlowState1 unit, DataFlow::Node sizeSource, FlowState2 extra
```

Comment:
Maybe just `size` or `allocSize` instead of `extra`?

Reply:
It's not really the size, though. The size of the allocation is `sizeSource + extra`. But if you're happy with `sizeSource + size` being the size of the allocation then I'm also fine with it 🙂.

Reply:
Ah, you're right. But can we come up with something better than extra?

Reply:
Hmm... How about `addend`?

Reply:
`sizeAddend`?

Reply:
Sold!

Reply:
c44507cc42c0b74fc541db6bf2b11b5a01440c6f

![image](https://github.com/github/codeql/assets/4527323/4d674289-d699-42fa-81f2-db1e744de9b5)